service-discovery/README.adoc

```diff
@@ -31,304 +31,399 @@ WARNING: This content is generated by running npm --prefix .build run generate:a
+== Installing the RHOAS Operator on OpenShift
+
+[role="_abstract"]
+Before you can bind Kafka instances in {product-long} to applications in OpenShift, you need to install the Red Hat OpenShift Application Services (RHOAS) Operator. The following procedure shows how to use the OperatorHub interface in the OpenShift web console to install the RHOAS Operator.
 
 .Prerequisites
-* You have bash/shell environment that can be used to download and install required tools
+* You have cluster administrator access for your OpenShift cluster.
 
 .Procedure
-. Install oc command line by following https://docs.openshift.com/container-platform/4.7/cli_reference/openshift_cli/getting-started-cli.html#installing-openshift-cli[official guide]
-. Install RHOAS CLI by downloading binaries from the latest release https://github.com/redhat-developer/app-services-cli/releases/latest
+. Log in to the OpenShift web console as a cluster administrator.
+. Click the perspective switcher in the top-left corner. Switch to the *Administrator* perspective.
+. In the left menu, click *Operators* > *OperatorHub*.
+. In the *Filter by keyword* field, enter `RHOAS`.
+. Select the *OpenShift Application Services (RHOAS)* Operator.
+. If you see a dialog box entitled *Show community Operator*, review the included information. When you have finished, click *Continue*.
++
+An information sidebar for the RHOAS Operator opens.
+
+. In the sidebar, review the information about the RHOAS Operator and click *Install*.
+. On the *Install Operator* page:
+.. For the *Installation mode* option, ensure that `All namespaces on the cluster` is selected.
+.. For the *Update channel* and *Update approval* options, keep the default values.
+.. Click *Install*.
+. When the installation process finishes, click *View Operator* to see the RHOAS Operator details. Alternatively, in the left menu, click *Operators* > *Installed Operators* and then click the RHOAS Operator.
++
+Observe that the RHOAS Operator is installed in the `openshift-operators` namespace by default.
 
-ifdef::qs[]
-.Verification
-* Do you see version printed when running `oc version` command in your terminal
-* Do you see version printed when running `rhoas version` command in your terminal
-endif::[]
+[id="proc-binding-openshift-to-{product-long}-using-cli_{context}"]
+== Tutorial: Binding an OpenShift application to {product-long} using the RHOAS CLI
 
-[id="proc-installing-operator_{context}"]
-== Installing the RHOAS Operator in your OpenShift cluster
+[role="_abstract"]
+When the Red Hat OpenShift Application Services (RHOAS) Operator is installed in your OpenShift cluster, you can use the RHOAS CLI to interact directly with the cluster. For example, you can use the CLI to connect an application running in the cluster to a Kafka instance in {product-long}. Establishing a connection between an OpenShift application and a backing service such as {product} is called _service binding_.
```

Comment:
This possibly makes the Command Line Interface sound like a general purpose tool for interacting with the cluster when in fact it's very specific to connecting to the Kafka service. Your "for example" is the *only* thing it can do with the cluser.